src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -1248,18 +1254,6 @@ public GridSetBroker getGridSetBroker() {
         return gridSetBroker;
     }
 
-    // public GeoServerTileLayer getTileLayerById(final String id) {
-    // return embeddedConfig.getLayerById(id);
-    // }
-
-    public List<LayerInfo> getLayerInfos() {
-        return getCatalog().getLayers();
-    }
-
-    public List<LayerGroupInfo> getLayerGroups() {
-        return getCatalog().getLayerGroups();
-    }
-
```

Comment:
The idea is that these methods are dangerous, when used the trigger lots of db loads.  I can add them back as deprecated, no problem, I removed only because I wanted to make sure not to leave anything using them.
And then I left them removed because the GWC facade is, to the limits of my knowledge, not used anywhere outside of the GWC module.